app/Jobs/DeleteQueryserviceNamespaceJob.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace App\Jobs;
+
+use App\QueryserviceNamespace;
+use App\Http\Curl\CurlRequest;
+use App\Http\Curl\HttpRequest;
+use Illuminate\Contracts\Queue\ShouldBeUnique;
+
+class DeleteQueryserviceNamespaceJob extends Job implements ShouldBeUnique
+{
+    private $wikiId;
+
+    /**
+     * @return void
+     */
+    public function __construct( $wikiId )
+    {
+        $this->wikiId = $wikiId;
+    }
+
+    /**
+     * The unique ID of the job.
+     *
+     * @return string
+     */
+    public function uniqueId()
+    {
+        return strval($this->wikiId);
+    }
+
+    /**
+     * @return void
+     */
+    public function handle( HttpRequest $request )
+    {
+        $qsNamespace = QueryserviceNamespace::whereWikiId($this->wikiId)->first();
```

Comment:
We could / should check the delete flag here as in the elastic delete job

```
$wiki = Wiki::withTrashed()->where( [ 'id' => $this->wikiId ] )->with('settings')->with('wikiDb')->first();
```

Reply:
done